### ISSUE TITLE
Ambiguous Regex reference on install

### ISSUE THREAD

**Issue by ashiklom** [linvinus/AltYo#67]
Trying to install this on Arch Linux (Gnome 3.22.1), got the following error on compilation:

```
altyo_terminal.vala:251.30-251.34: error: `Regex' is an ambiguous reference between `GLib.Regex' and `Vte.Regex'
              if(!done[0] && Regex.match_simple(".*_REPLACE_.*",this.tab_title_regex[i+1])){
                             ^^^^^
altyo_terminal.vala:257.30-257.34: error: `Regex' is an ambiguous reference between `GLib.Regex' and `Vte.Regex'
              if(!done[1] && Regex.match_simple(".*_USER_.*",this.tab_title_regex[i+1])){
                             ^^^^^
altyo_terminal.vala:263.30-263.34: error: `Regex' is an ambiguous reference between `GLib.Regex' and `Vte.Regex'
              if(!done[2] && Regex.match_simple(".*_HOSTNAME_.*",this.tab_title_regex[i+1])){
                             ^^^^^
altyo_terminal.vala:270.30-270.34: error: `Regex' is an ambiguous reference between `GLib.Regex' and `Vte.Regex'
              if(!done[3] && Regex.match_simple(".*_PATH_.*",this.tab_title_regex[i+1])){
                             ^^^^^
altyo_terminal.vala:1065.53-1065.57: error: `Regex' is an ambiguous reference between `GLib.Regex' and `Vte.Regex'
        var key=this.vte_term.match_add_gregex((new Regex (url_regexps[j])),0);
                                                    ^^^^^
Compilation failed: 5 error(s), 0 warning(s)
make: *** [Makefile:79: default] Error 1
```

Seems there are two competing packages, both with `Regex` functions. Seems like an easy fix that I'd be happy to implement myself if you could just specify which of those packages is preferred (I know next to nothing about Vala...).

**Comment by linvinus** [comment]
hi, please try GLib.Regex

**Comment by ashiklom** [comment]
Fixed in #68 .
